examples/name-service-without-macros/programs/name-service/src/lib.rs

```diff
@@ -0,0 +1,222 @@
+use std::net::{Ipv4Addr, Ipv6Addr};
```

Comment:
btw we can put multiple programs into one anchor project then we don't have all these boilerplate files

Reply:
Good idea, will do